core/src/CrossOrigin.ts

```diff
@@ -347,6 +347,7 @@ interface CrossOriginElementDataIn {
     rootId?: string;
     ownerId?: string;
     observedName?: string;
+    accessible?: boolean;
```

Comment:
Not very clear what `accessible` is supposed to represent here, let's ad jsdoc and consider a rename to smth that makes more sense